### PR TITLE
[CI] Format json

### PR DESCRIPTION
**Pull request by dayo09**
Resolves https://github.com/Samsung/ONE-vscode/issues/359

### CODE REVIEW

**Comment by dayo09** [comment]
### Test

With unformatted file, [2401816](https://github.com/Samsung/ONE-vscode/pull/398/commits/2401816947b83917c0b8ffcd64b1e645610683f3) `Check PR Format` failed.

https://github.com/Samsung/ONE-vscode/actions/runs/2027428645

Generated format.patch:
```
diff --git a/src/Editor/json/tools_attr.json b/src/Editor/json/tools_attr.json
index a0907a5..37f4220 100644
--- a/src/Editor/json/tools_attr.json
+++ b/src/Editor/json/tools_attr.json
@@ -245,7 +245,7 @@
       {
         "attr_name": "fuse_add_with_fully_connected",
         "attr_desc": "fuse Add op to FullyConnected op"
-      }  ,
+      },
       {
         "attr_name": "fuse_batchnorm_with_conv",
         "attr_desc": "fuse BatchNorm op to Convolution op"
```

**Review by llFreetimell — APPROVED** [review]
LGTM

File: infra/format [hunk 1]
```diff
@@ -109,6 +109,31 @@ function check_ts_files() {
...
+    if [[ ${f} == +(*.json) ]]; then
+      FILES_TO_CHECK_JSON+=("${f}")
+    fi
+  done
+  
+  for f in ${FILES_TO_CHECK_JSON[@]}; do
+    jq . ${f} > ${f}.temp
+    rm ${f} && mv ${f}.temp ${f}
+  done
+
```
Comment (mhs4670go) [inline infra/format:134]: nit.
```suggestion
```

**Review by mhs4670go — APPROVED** [review]
LGTM

**Review by llFreetimell — APPROVED** [review]
LGTM

**Pull request merged**